### PR TITLE
Fix two cbmc tests

### PR DESCRIPTION
**Pull request by kroening**

### CODE REVIEW

File: regression/cbmc/Typecast2/main.i [hunk 1]
```diff
@@ -0,0 +1,7 @@
+int main()
+{
+  unsigned int i=2;
+  __CPROVER_assert(0l==(signed long int)(i - (unsigned int)2),
+                   "difference of cast");
```
Comment (tautschnig) [inline regression/cbmc/Typecast2/main.i:5]: Why is this change necessary and/or beneficial?

Comment (kroening) [reply regression/cbmc/Typecast2/main.i:5]: When run on a 32-bit system, you would otherwise need a cross-compiler to 64-bit.

**Review by tautschnig — APPROVED** [review]

File: regression/cbmc/Typecast2/main.i (hunk 1 above)
Comment (tautschnig) [reply regression/cbmc/Typecast2/main.i:5]: Oh, of course, I'm sorry for having been dense. Yet it would be nice if such information were included in the commit message.

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: bc7ab4ec638d3f3d563b369ec5c7fe130b587590).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/76730436

File: regression/cbmc/Typecast2/main.i (hunk 1 above)
Comment (kroening) [reply regression/cbmc/Typecast2/main.i:5]: Ok, now in commit message.

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: bcc3bef4a44a7fd9dd26a1fa76d7846fcd674ba4).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/76735329
